resources/views/admin.blade.php

```diff
@@ -55,7 +55,7 @@
                             <select id="select" id="company" name="company_id" class="form-select form-control m-2 " aria-label="Default select example">
                                 <option hidden>Company</option>
                                 @foreach ( $companies  as $company )
-                                    <option value="{{ $company->id }}" class="create-option" data-branch="{{ $company->branch }}"> {{ $company->name }} </option>
+                                    <option value="{{ $company->id }}" id="option-id-{{ $company->id }}" class="create-option" data-branch="{{ $company->branch }}"> {{ $company->name }} </option>
```

Comment:
please do not set dynamic identifier on static forms. This should be used when we clone form. please refector the code to setup a single unique ID like `company_id`

Reply:
sure ill refactor the code set up a single unique id

Reply:
removed the dynamic ids and options set using the value please review @Raza9798